backend/controllers/authController.js

```diff
@@ -0,0 +1,24 @@
+const User = require('/models/UserModels');
+
+const authController = {
+  async getUser(req, res, next) {
+    try {
+      const data = await User.findOne({ firstName });
+    } catch (e) {
+      console.log(`Error: ${e}`);
+    }
+    console.log(`Here is the data: ${data}`);
+  },
+
+  async signUp(req, res, next) {
+    try {
+      const { username } = req.body;
+      const newUser = User.create({ username });
+    } catch (e) {
+      console.log(`Error: ${e}`);
+    }
+    console.log(`Here is the new user: ${newUser}`);
```

Comment:
I would also pass the user details through `res.locals` so the next middleware in the chain has access to it

Reply:
Thanks for the advice, I've made adjustments and recommitted.